src/uwtools/utils/file_helpers.py

```diff
@@ -1,54 +1,37 @@
 """
-Helpers to be used when working with files and directories
+Helpers for working with files and directories
 """
 
 import logging
 import os
-from datetime import datetime
 import shutil
+from datetime import datetime as dt
 
 
-def handle_existing(run_directory, exist_act): #pylint: disable=unused-variable
-    '''Given a run directory, and an action to do if
-    directory exists, delete or rename directory.'''
+def handle_existing(run_directory: str, exist_act: str) -> None:
+    """
+    Given a run directory, and an action to do if directory exists, delete or
+    rename directory.
+    """
 
-    logging.getLogger(__name__)
+    # Try to delete existing run directory if option is delete.
 
-    # Try to delete existing run directory if option is delete
     try:
         if exist_act == "delete" and os.path.isdir(run_directory):
             shutil.rmtree(run_directory)
-    except (RuntimeError, FileExistsError) as del_error:
+    except (FileExistsError, RuntimeError) as e:
         msg = f"Could not delete directory {run_directory}"
         logging.critical(msg)
-        raise RuntimeError(msg) from del_error
+        raise RuntimeError(msg) from e
+
+    # Try to rename existing run directory if option is rename.
 
-    # Try to rename existing run directory if caller chooses rename
     try:
         if exist_act == "rename" and os.path.isdir(run_directory):
-            now = datetime.now()
-            save_dir = run_directory + now.strftime("_%Y%m%d_%H%M%S")
+            now = dt.now()  # #PM# SHOULD WE UTCNOW?
```

Comment:
I think in this case, it might be helpful to users to have this particular thing adopt the time zone used by the platform where it's being run. It is only meant to be a unique string added to the path when we move it out of the way.